packages/firestore/src/remote/serializer.ts

```diff
@@ -1090,8 +1090,8 @@ export class JsonProtoSerializer {
     } else if (filter.fieldFilter !== undefined) {
       return [this.fromRelationFilter(filter)];
     } else if (filter.compositeFilter !== undefined) {
-      return filter.compositeFilter.filters!
-        .map(f => this.fromFilter(f))
+      return filter.compositeFilter
+        .filters!.map(f => this.fromFilter(f))
```

Comment:
I'm not sure why this code got touched?  A prettier fail somewhere?

Reply:
I think these problems show up for me when I ran yarn from master versus yarn from firestore-multi-tab. I re-ran it from firestore-multi-tab for this PR.